src/main/java/appeng/container/AEBaseContainer.java

```diff
@@ -117,6 +119,17 @@ public AEBaseContainer( final InventoryPlayer ip, final TileEntity myTile, final
 		this.prepareSync();
 	}
 
+	@Override
+	public void onCraftMatrixChanged( final IInventory par1IInventory )
+	{
+		this.onCraftMatrixChanged();
+	}
+
+	public void onCraftMatrixChanged()
+	{
+		this.detectAndSendChanges();
```

Comment:
Oh, this is from before i added the IItemHandler -> IInventory wrapper. I will change this back and pass a wrapped item handler to the method.